app/cp_nigeria/views.py

```diff
@@ -1199,43 +1198,43 @@ def cpn_outputs(request, proj_id, step_id=STEP_MAPPING["outputs"], complex=False
     cash_flow.loc["DSCR"] = cash_flow.loc["Cash flow from operating activity"] / (
         losses.loc["Equity interest"] + losses.loc["Debt interest"] + senior_debt.loc["Principal"]
     )
-    financial_kpis = ft.financial_kpis
-    # calculate the financial KPIs with 0% grant
-    ft.remove_grant()
-    no_grant_tariff = ft.tariff
-    no_grant_kpis = ft.financial_kpis
-
-    comparison_kpi_df = pd.DataFrame([financial_kpis, no_grant_kpis], index=["With grant", "Without grant"]).T
-    comparison_kpi_df.loc["Estimated tariff per kWh"] = {
-        "With grant": tariff * ft.exchange_rate,
-        "Without grant": no_grant_tariff * ft.exchange_rate,
-    }
-
-    help_texts = {
-        "Total investment costs": help_icon("All upfront investment costs"),
-        "Total equity": help_icon("Total equity help text"),
-        "Total grant": help_icon("Total grant help text"),
-        "Initial loan amount": help_icon("Initial loan needed to cover the initial investment costs"),
-        "Replacement loan amount": help_icon("Amount needed to replace the system components"),
-    }
+    # financial_kpis = ft.financial_kpis
```

Comment:
it is better to remove code which will not be used, we have version control to reverse the change if needbe